setup.py

```diff
@@ -32,9 +32,10 @@
     author='Jannis Gebauer',
     author_email='support@pyup.io',
     url='https://github.com/pyupio/safety-db/',
-    py_modules=['safety_db'],
     license='Attribution-NonCommercial-ShareAlike 4.0 International',
     install_requires=[],
     tests_require=tests_require,
     include_package_data=True,
+    packages=['safety_db'],
```

Comment:
My best guess is that the original author had a point on making this Python package simple, even though I like your proposal of creating a package just for this. Even more if that is the underlying reason behind the issue.